lib/resolver.content.js

```diff
@@ -120,15 +120,20 @@ module.exports = class PodletClientContentResolver {
                 headers,
             };
 
-            const timer = this.metrics.timer({
-                name: 'podlet_content_request',
+            const histogram = this.metrics.histogram({
+                name: 'podium_client_resolver_content_resolve',
                 description:
                     'Time taken for success/failure of content request',
-                meta: {
-                    url: reqOptions.uri,
-                    method: reqOptions.method,
+                labels: {
+                    url: outgoing.fallbackUri,
```

Comment:
This is the resolver which fetches the content. I think using fallbackUri here is wrong. Should this be `outgoing.contentUri`?

Reply:
shooot, copypasta, good catch